doc/source/tune/_tutorials/overview.rst

```diff
@@ -73,6 +79,7 @@ Take a look at any of the below tutorials to get started with Tune.
    tune-sklearn.rst
    tune-pytorch-cifar.rst
    tune-pytorch-lightning.rst
+   tune-serve-integration-mnist.rst
```

Comment:
I don't see this `.rst` file - what am I missing?

Reply:
That's the [sphinx-gallery](https://sphinx-gallery.github.io/stable/syntax.html) plugi at work. Basically it takes a formatted python file (`_tutorials/tune-serve-integration-mnist.py`) and creates the RST and an ipython notebook from it. Running `sphinx-build` will first generate these files and place them into `tutorials` (without the underscore), where it is then found by `overview.rst`.

Reply:
We already do this [with the sklearn tutorial](https://docs.ray.io/en/master/tune/tutorials/tune-sklearn.html).